mmselfsup/core/hooks/momentum_update_hook.py

```diff
@@ -5,11 +5,11 @@
 from mmcv.runner import HOOKS, Hook
 
 
-@HOOKS.register_module()
-class BYOLHook(Hook):
-    """Hook for BYOL.
+@HOOKS.register_module(name=['BYOLHook', 'MomentumUpdateHook'])
+class MomentumUpdateHook(Hook):
```

Comment:
Better name as `MomentumUpdaterHook`

Reply:
Actually, mmcv has hooks called `EMAHook` and `MomentumUpdaterHook`.